src/styles/stars.scss

```diff
@@ -0,0 +1,30 @@
+.stars-rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+
+  &__review {
+    color: #060b35;
```

Comment:
Consider defining the color `#060b35` as an SCSS variable in the separate variables file to maintain consistency and ease of maintenance.